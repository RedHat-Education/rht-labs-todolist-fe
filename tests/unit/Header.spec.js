import { shallow } from "@vue/test-utils";
import Header from '@/components/Header.vue'
import * as all from "../unit/setup.js";

describe("Header.vue", () => {
    it("has the expected html structure", () => {
        const wrapper = shallow(Header);
        expect(wrapper.element).toMatchSnapshot();
    })
})