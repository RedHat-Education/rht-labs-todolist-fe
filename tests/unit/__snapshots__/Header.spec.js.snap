// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.vue has the expected html structure 1`] = `
<div
  class="todo-header"
>
  <div
    class="overlay"
  />
   
  <time
    class="clearfix"
  >
    <span
      class="day"
    >
      10
    </span>
     
    <span
      class="dayofweek"
    >
      Tuesday
    </span>
     
    <span
      class="dayofweek"
    >
      April, 2018
    </span>
  </time>
</div>
`;
